### ISSUE TITLE
feat(eslint): Add a11y plugin to eslint

### ISSUE THREAD

**Issue by scarletmerlin123** [OpenCTI-Platform/opencti#16818]
## Use case

Update eslint to use [eslint-plugin-jsx-a11y](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y). This would help contributors catch some accessibility issues. 

## Current workaround

Currently it is up to the contributor to manually review for missing or incorrect aria props

## Proposed solution

add eslint-plugin-jsx-a11y as a dev dependency and add it to the eslint config

## If the feature request is approved, would you be willing to submit a PR?

Yes
